### PR TITLE
fastdoubleparser 1.0.0

### PR DESCRIPTION
**Pull request by pjfanning**
relates to #1161 and https://github.com/wrandelshofer/FastDoubleParser/pull/77

### CODE REVIEW

**Comment by pjfanning** [comment]
@cowtowncoder this should fox the NPE in #1161 

I would like to backport this to 2.16.1 if that's ok. Users who enable fast parsing of big decimals could easily hit the NPE.

File: pom.xml [hunk 1]
```diff
@@ -182,6 +182,7 @@ com.fasterxml.jackson.core.*;version=${project.version}
                   <artifact>ch.randelshofer:fastdoubleparser</artifact>
                   <excludes>
                     <exclude>META-INF/versions/**/module-info.*</exclude>
+                    <exclude>META-INF/versions/22/**/*.*</exclude>
```
Comment (pjfanning) [inline pom.xml:185]: due to https://github.com/wrandelshofer/FastDoubleParser/issues/79

**Comment by cowtowncoder** [comment]
> @cowtowncoder this should fox the NPE in #1161
> 
> I would like to backport this to 2.16.1 if that's ok. Users who enable fast parsing of big decimals could easily hit the NPE.

I was about to suggest the same, +1 for backport to 2.16.1.

**Review by cowtowncoder — APPROVED** [review]

**Comment by cowtowncoder** [comment]
Merged to `master` (part of it manual due to ignoral of pom.xml merge failures).

**Pull request merged**